Doc/library/netrc.rst

```diff
@@ -32,6 +32,11 @@ the Unix :program:`ftp` program and other FTP clients.
 
    .. versionchanged:: 3.4 Added the POSIX permission check.
 
+   .. versionchanged:: 3.7
+      Now uses :func:`os.path.expanduser`, so if :envvar:`HOME` is not
```

Comment:
I think the "if :envvar:`HOME` is not set [...]" part should be moved to the main documentation block and the versionchanged annotation should only mention that it now uses expanduser() to determine the location of the .netrc file if \*file\* is not passed.